opencti-platform/opencti-front/src/private/components/settings/activity/alerting/Alerting.tsx

```diff
@@ -215,12 +217,18 @@ const Alerting: FunctionComponent = () => {
   };
 
   return (
-    <div className={classes.container}>
-      <ActivityMenu/>
-      <Breadcrumbs variant="list" elements={[{ label: t_i18n('Settings') }, { label: t_i18n('Activity') }, { label: t_i18n('Alerting'), current: true }]} />
-      {renderLines()}
-      <AlertCreation paginationOptions={paginationOptions}/>
-    </div>
+    <Security needs={[SETTINGS_SECURITYACTIVITY]} placeholder={<span>{t_i18n(
```

Comment:
why do we need this ? since the component is already protected in the root. Do we import it elsewhere ? in which cases do we display this message ?